kong/init.lua

```diff
@@ -746,8 +746,9 @@ function Kong.init()
   require("resty.kong.var").patch_metatable()
 
   if config.dedicated_config_processing and is_data_plane(config) then
-    -- TODO: figure out if there is better value than 2048
-    local ok, err = process.enable_privileged_agent(2048)
+    -- TODO: figure out if there is better value than 4096
```

Comment:
Have we got a chance to address the TODO?

Reply:
no idea

Reply:
@bungle What do you think?